Run cpp exceptions test

- I guess it requires patching emscripten?

```
_____________________ test_cpp_exception_catching[chrome] ______________________
packages/cpp-exceptions-test/test_cpp_exceptions.py:34: in test_cpp_exception_catching
    assert (
/usr/share/miniconda/envs/pyodide-env/lib/python3.11/site-packages/pytest_pyodide/runner.py:260: in run_js
    return self.run_js_inner(code, check_code)
/usr/share/miniconda/envs/pyodide-env/lib/python3.11/site-packages/pytest_pyodide/runner.py:365: in run_js_inner
    raise JavascriptException(retval[1], retval[2])
E   pytest_pyodide.runner.JavascriptException: 9808840
----------------------------- Captured stdout call -----------------------------
JavascriptException message:  None
--------------------------- Captured stdout teardown ---------------------------
cpp-exceptions-test already loaded from default channel
No new packages to load
______________________________ test_sjlj[chrome] _______________________________
packages/cpp-exceptions-test/test_cpp_exceptions.py:60: in test_sjlj
    assert (
/usr/share/miniconda/envs/pyodide-env/lib/python3.11/site-packages/pytest_pyodide/runner.py:260: in run_js
    return self.run_js_inner(code, check_code)
/usr/share/miniconda/envs/pyodide-env/lib/python3.11/site-packages/pytest_pyodide/runner.py:365: in run_js_inner
    raise JavascriptException(retval[1], retval[2])
E   pytest_pyodide.runner.JavascriptException: Error: Dynamic linking error: cannot resolve symbol setjmp
E       at stubs.<computed> (http://127.0.0.1:38383/pyodide.asm.js:9:140062)
E       at wasm://wasm/75[121](https://github.com/ryanking13/pyodide-recipes-mirror/actions/runs/6518437551/job/17704012132#step:8:122)092:wasm-function[8]:0x1d7
E       at run (eval at executeAsyncScript (:445:26), <anonymous>:9:17)
E       at async eval (eval at executeAsyncScript (:445:26), <anonymous>:13:34)
----------------------------- Captured stdout call -----------------------------
JavascriptException message:  Dynamic linking error: cannot resolve symbol setjmp
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -284,12 +284,11 @@ jobs:
       - name: Run tests
         shell: bash -l {0}
         run: |
-          # FIXKE: don't skip cpp_exception tests
           pytest -v \
             --dist-dir=./dist/ \
             --runner=${{ matrix.test-config.runner }} \
             --rt ${{ matrix.test-config.runtime }} \
-            -k "not cpp_exception and not triangulation and not cartopy" \
+            -k "not triangulation and not cartopy" \
             packages
   release:
     runs-on: ubuntu-latest

